src/core/credrank/markovProcessGraph.js

```diff
@@ -270,22 +274,22 @@ export class MarkovProcessGraph {
           owner: participant.id,
           epochStart: boundary,
         };
-        addEdge(payoutGadget.markovEdge(thisEpoch, beta));
+        recordTransitionProbability(payoutGadget.markovEdge(thisEpoch, beta));
         if (lastBoundary != null) {
           const webbingAddress = {
             thisStart: boundary,
             lastStart: lastBoundary,
             owner: participant.id,
           };
-          addEdge(
+          recordTransitionProbability(
             forwardWebbingGadget.markovEdge(webbingAddress, gammaForward)
           );
-          addEdge(
+          recordTransitionProbability(
             backwardWebbingGadget.markovEdge(webbingAddress, gammaBackward)
           );
         }
-        lastBoundary = boundary;
       }
+      lastBoundary = boundary;
```

Comment:
Note to reviewer (and to self): I believe this `lastBoundary` assignment was in the wrong loop, and this was causing the wrong fibration edges to get written. I believe that is why the Cred shifted slightly in this commit. For cleanliness and sanity checking, let's pull this change out into a preceding commit before merging.

Reply:
Done, see #2402.